R/write.R

```diff
@@ -75,12 +75,12 @@ xportr_write <- function(.df,
     abort("`.df` file name must be 8 characters or less.")
   }
 
+  checks <- xpt_validate(.df)
+
   if (stringr::str_detect(name, "[^a-zA-Z0-9]")) {
```

Comment:
can you make a comment in the code on why this is relaxed

Reply:
Great suggestion! I've made an update to New.md by adding a brief description.